app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEWallet.java

```diff
@@ -224,16 +225,18 @@ public VEEWallet(){
         seed = "";
         accountSeeds = new ArrayList<>();
         nonce = 0;
+        chainId = 0;
```

Comment:
Should it be initialized to 'M' here for consistency? As the value is among either 'M' or 'T' above